src/helpers/dom.js

```diff
@@ -220,179 +282,191 @@ export function crowdContribute(domain, path, target) {
  * @param {string} target - The target must be a fully qualified URL including the protocol
  */
 export function followAndContribute(domain, path, target) {
-  var data = {
-    domain: domain,
-    path: path,
-    target: target,
-  };
-  document.dispatchEvent(
-    new CustomEvent('ff53054c0e13_followAndContribute', { detail: data })
-  );
+    var data = {
+        domain: domain,
+        path: path,
+        target: target,
+    };
+    document.dispatchEvent(
+        new CustomEvent('ff53054c0e13_followAndContribute', {detail: data})
+    );
 }
-
+/**
+ * Will change the URL of the page directly by routing it through our servers. This is a more secure way to bypass a link shortener, but not as fast as unsafelyAssign. This also prevents the user's IP address from being leaked if the option is turned on in the extension settings.
```

Comment:
```suggestion
 * Will change the URL of the page by sending a request to fastforward.team which will be intercepted by the background script (or declarativeNetRequest rules on chromium) . This is a more secure way to bypass a link shortener, but not as fast as unsafelyAssign. This also prevents the user's IP address from being leaked if the option is turned on in the extension settings.
```

The fastforward.team/bypassed page is actually just a placeholder with an error, the redirecting is done by the bg script/declarativeNetRequest rule set